popup.js

```diff
@@ -0,0 +1,53 @@
+function msToTime(s) {
+    var ms = s % 1000;
+    s = (s - ms) / 1000;
+    var secs = s % 60;
+    s = (s - secs) / 60;
+    var mins = s % 60;
+    var hrs = (s - mins) / 60;
+    if (secs < 10)
+        secs = '0' + secs;
+    if (mins < 10)
+        mins = '0' + mins;
+
+    return hrs + ':' + mins + ':' + secs;
+}
+
```

Comment:
Use http://www.w3schools.com/jsref/jsref_obj_date.asp instead of manual math.
Something like:
var date = new Date();
date.setMilliseconds(ms);
date.toISOString().substr(11, 8);